pages/blog/march-2023.md

```diff
@@ -0,0 +1,52 @@
+authors:
+  - name: Thulisile Sibanda
+    photo: /img/avatars/thulieblack.webp
+    link: https://www.linkedin.com/in/v-thulisile-sibanda/
+    byline: AsyncAPI Community Manager
+excerpt: 'Community update for March'
+featured: true
+---
+March came with some ecstatic news for the AsyncAPI community. We welcomed [Thulisile Sibanda](https://www.linkedin.com/in/v-thulisile-sibanda/) as our new Community Manager. And to top it all, our beloved community guardian, [Lukasz Gornicki](https://www.linkedin.com/in/lukasz-gornicki-a621914/) stepped up to become the new [Executive Director](https://github.com/orgs/asyncapi/discussions/615) for AsyncAPI.
+This is such an incredible milestone for the community and we can't wait to see the amazing things to follow in the next few months.
+
+## AsyncAPI Conf on Tour 2023
+One of our goals this year is to host [AACoT](https://opencollective.com/asyncapi/events/asyncapi-conference-on-tour-6b3c0aa1), a series of micro-conferences in different locations around the world. We are glad that our first location has been confirmed and we'll be in Madrid on the 18th - 19th of October 2023. [Call for speakers](https://sessionize.com/aacot-madrid) will be open soon and we are eagerly waiting to receive your submissions. For more details about other AACoT locations, keep an eye out for this [discussion](https://github.com/orgs/asyncapi/discussions/598).
+
+## Case Studies
+We recently added a new [case studies](https://www.asyncapi.com/casestudies) section on our website. It showcases real-world production scenarios and how different individuals adopt and integrate our specifications and tools. On this note, we are glad to have our first official case study from Adeo featured on the website. You can read more about it [here](https://www.asyncapi.com/casestudies/adeogroup).
+
+## Spec x Tooling
+[Modelina](https://modelina.org/) website has been merged and if you are interested in generating data models for your payloads, do try it out.
+Spec 3.0 meetings will now incorporate various discussions and will be the best place to ask about v3, how to use [non-JSON schemas](https://github.com/asyncapi/spec/pull/910) within AsyncAPI, and [documentation](https://github.com/asyncapi/website/issues/1433).
+
+
+## Google Summer of Code
+As a community, we are always looking for initiatives to drive diversity and bring more people passionate about open-source. In this regard, we applied to GSoC'23. Although our application didn't make it to the final organization selection, we are privileged to participate in GSoC'23 under Postman. 
+
+Contributor applications open on the 20th of March and close on the 4th of April. If you are looking to participate we have some exciting [projects](https://github.com/postman-open-technologies/gsoc-2023/issues?q=is%3Aissue+is%3Aopen+label%3Afinal+label%3Aasyncapi+) and if you want to make your proposal outshine do watch this [session](https://www.youtube.com/live/ECe592525p4?feature=share) for tips.
```

Comment:
```suggestion
Contributor applications open on the 20th of March and close on the 4th of April. If you are looking to participate we have some exciting [AsyncAPI projects](https://github.com/postman-open-technologies/gsoc-2023/issues?q=is%3Aissue+is%3Aopen+label%3Afinal+label%3Aasyncapi+) and if you want to make your proposal outshine do watch this [session](https://www.youtube.com/live/ECe592525p4?feature=share) for tips.
```

For accessibility, remember to always write a descriptive anchor text. Consider that "project" is too generic if read out by a screen reader.